app/models/concerns/custom_field_addable.rb

```diff
@@ -118,6 +124,12 @@ def custom_field?(field_identifier)
     self.class.custom_field?(field_identifier, model: self)
   end
 
+  def unanswered?(field_identifier)
+    value = custom_value(field_identifier)
+    field = custom_field(field_identifier)
+    value.blank? && field.descendants.all? { |i| custom_value(i.id).blank? }
+  end
```

Comment:
What changes were you thinking of making? I've been kicking myself for ever thinking modeling this system using custom fields was a good idea. I think we should've just created LoanQuestion, LoanResponse, LoanQuestionSet and LoanResponseSet. If we needed custom fields for "is recovered", we could still create them and they could be way simpler. But that would be a huge refactor.